test/evergreen.yml

```diff
@@ -154,6 +154,25 @@ tasks:
           configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ CFLAGS="-fsanitize=undefined -ggdb"
           posix_configure_flags: --enable-silent-rules --enable-strict --enable-diagnostic
 
+  # build with --disable-shared, without --enable-diagnostic
+  # the compiler builds turn on diagnostics
```

Comment:
This comment confused me. Lets clarify it with team.

Reply:
updated